src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -170,6 +170,19 @@ private static ISymbol MapSymbol(ISymbol symbol, ITypeSymbol type)
                 }
             }
 
+            if (symbol is IMethodSymbol methodSymbol &&
```

Comment:
I'll admit it's 100% unclear to me what this method in general is doing. It seems to be a smorgasboard of random uncommented stuff. Since I'm guessing you _do_ (or else you wouldn't have modified it), can you stick a doc comment on what it's supposed to do?

Reply:
Will do.